PHPCI/Plugin/CopyBuild.php

```diff
@@ -75,7 +75,7 @@ public function execute()
     protected function wipeExistingDirectory()
     {
         if ($this->wipe === true && $this->directory != '/' && is_dir($this->directory)) {
-            $cmd = 'rm -Rf "%s*"';
+            $cmd = 'rm -Rf "%s"*';
```

Comment:
for explanation:
in my opinion the whole topic about the wildcard is as follows:
The original command rm -Rf "%s*" did not work, because the %s is escaped and not interpreted as wildcard.
I recognized, that even if the wildcard would be interpreted correctly, $this->directory NEEDS a trailing slash to wipe all content inside the specified directory.
So I fixed the command to work properly by moving te wildcard out of the quotes.

I think the final step is to make sure, that $this->directory always has a trailing slash.

What do you think?